docs/development/releasing.rst

```diff
@@ -442,6 +460,46 @@ be fixed soon to a new release milestone.  If the upcoming bug fix release is
 issues that you don't expect to be fixed in time for 'v0.2.2'.
 
 
+.. _helpers-release-info:
+
+Coordinating Astropy and astropy-helpers Releases
+-------------------------------------------------
+
+A bit more initial effort is required for an Astropy release that has a
+corresponding astropy-helpers release.  The main reason for this more complex
+procedure is to allow the Astropy core to be tested aginst the new helpers
+before anything is released.  Hence the following procedure should be added
+to the beginning of the above procedure when this is required.
+
+#. In `astropy-helpers`, create a new release branch "release-<version>".
+
+#. Create the release commit (updating the version info and changelog) in that
+   branch.
+
+#. Push the release branch to github.
+
+#. In astropy master (and/or the relevant maintainence branch), issue a PR
+   updating the helpers to the commit described in the last step (i.e., the
+   head of the astropy-helpers release branch).
+
+#. Wait for Travis to run to ensure that helpers build works with Astropy.
+   If it doesn't, back out the release and fix whatever the problem is before
+   trying again.
+
+#. Assuming it does succeed, finish the release of the helpers by doing this in
+   the helpers repo::
+
+      git checkout master
+      git merge --no-ff release-<version>
+      git tag -s "v<version>" -m "Tagging v<version>"
```

Comment:
The merge also has to be done into the maintenance branch. Do we tag releases in master or in the maintenance branch?